diff --git a/CMake/folly-deps.cmake b/CMake/folly-deps.cmake
--- a/CMake/folly-deps.cmake
+++ b/CMake/folly-deps.cmake
@@ -167,7 +167,7 @@
 list(APPEND FOLLY_INCLUDE_DIRECTORIES ${LIBAIO_INCLUDE_DIRS})
 
 find_package(LibUring)
-set(FOLLY_HAS_LIBURING ${VCPKG_LOCK_FIND_LibUring})
+set(FOLLY_HAS_LIBURING ${VCPKG_LOCK_FIND_PACKAGE_LibUring})
 list(APPEND FOLLY_LINK_LIBRARIES ${LIBURING_LIBRARIES})
 list(APPEND FOLLY_INCLUDE_DIRECTORIES ${LIBURING_INCLUDE_DIRS})
 
diff --git a/folly/io/async/IoUringZeroCopyBufferPool.cpp b/folly/io/async/IoUringZeroCopyBufferPool.cpp
--- a/folly/io/async/IoUringZeroCopyBufferPool.cpp
+++ b/folly/io/async/IoUringZeroCopyBufferPool.cpp
@@ -27,41 +27,6 @@
 
 #if FOLLY_HAS_LIBURING
 
-// TODO(davidhwei): Remove once liburing catches up and gets synced with
-// fbsource.
-#define IORING_REGISTER_ZCRX_CTRL 36
-
-enum zcrx_reg_flags {
-  ZCRX_REG_IMPORT = 1,
-};
-
-enum zcrx_ctrl_op {
-  ZCRX_CTRL_FLUSH_RQ,
-  ZCRX_CTRL_EXPORT,
-
-  __ZCRX_CTRL_LAST,
-};
-
-struct zcrx_ctrl_flush_rq {
-  __u64 __resv[6];
-};
-
-struct zcrx_ctrl_export {
-  __u32 zcrx_fd;
-  __u32 __resv1[11];
-};
-
-struct zcrx_ctrl {
-  __u32 zcrx_id;
-  __u32 op; /* see enum zcrx_ctrl_op */
-  __u64 __resv[2];
-
-  union {
-    struct zcrx_ctrl_export zc_export;
-    struct zcrx_ctrl_flush_rq zc_flush;
-  };
-};
-
 namespace folly {
 
 class IoUringZeroCopyBufferPoolImpl {
